### DISCUSSION TITLE
Test server connection impossible since 05/09 TSO infrastructure maintenance

### DISCUSSION THREAD

**Discussion by Maetel2002** [fedorovvl/tso_client#307] [General]
Hi All !
After a maintenance on the 5th of September, until today, it remains impossible to log in on the Test Server with the Client.

First, we were able to connect with Unity, but with missing chats. So, TSO had to correct chat servers adresses because they were lost.
Now, Chat servers are back and Unity is working "normally" under the test server.

But it remains totally impossible to log in with the Client. 
We pass the authentification. Then, blue screen, Hammerman & impossible to connect.
![image](https://github.com/fedorovvl/tso_client/assets/131248724/0c45aa48-3ee9-4264-bbb4-3dca67f35bdb)
![image2](https://github.com/fedorovvl/tso_client/assets/131248724/31bd20d6-2dee-461c-88c2-9b71a8c48b02)

There's no version change.

On Production countries servers (FR & European English), Client still to be working properly.

Could you please investigate, because, if TSO has found an option to avoid any connection with the Client, confirm its experimentation on the test server, and extend it on Countries production servers with a next infrastructure maintenance, we will have a huge issue...

Thanks in advance for your feedbacks.

**Comment by Cdrwbw** [comment]
What's new
![image](https://github.com/fedorovvl/tso_client/assets/111175316/87659866-ca34-4d05-8fd4-621251022f8c)
Brazilian server  home page
![image](https://github.com/fedorovvl/tso_client/assets/111175316/26776ddf-7e4b-463e-8686-90e095375270)
Test server homepage
![image](https://github.com/fedorovvl/tso_client/assets/111175316/aac4d049-e28e-4cf5-93ed-f908e153f29a)
German server home page
![image](https://github.com/fedorovvl/tso_client/assets/111175316/384a6fdc-4f07-4ad2-8282-2296450a4eba)
EU server home page
the first two images show the addition of two "Sandbox eval code" scripts, which are not on the other servers, which still have the client running.
At this time the french server is not running but
![image](https://github.com/fedorovvl/tso_client/assets/111175316/19db0d98-90a6-483f-8ced-1f6318847f3b)
client blocking is in sight because of the sandboxes

**Reply by bellejt** [reply]
dutch server already involved.Lots off players are stopping

**Reply by Cdrwbw** [reply]
be patient, client is ok but not chats and not green dot in the friends/guilds player ribbon.
try latest build

**Reply by bellejt** [reply]
started client but still server unreachable on nederland. Downloaded from site but not working.Maybe link to good file for independant client ? When starting client this morning it updated and then we all got error

**Reply by Cdrwbw** [reply]
ok, Download the swf file here ==> https://drive.google.com/file/d/1-1T2jvYZjkxqS_mCWCJOKGGiLh0itvW-/view
in the tso_portable folder, in the shortcut used to launch the client, add --skip to the command line
it should be something like this "c:\folderofclient\client.exe" --skip

**Reply by bellejt** [reply]
thats working for portable.THX : but not for online in browser

**Reply by Cdrwbw** [reply]
copy the content folder of tso_portable in folder ubisoft/the Settlers Online and rename the client.exe in The Settlers Online.exe. make a copy of folder The Settlers Online before.
But why use client with browser ?

**Reply by bellejt** [reply]
Is not for my but for a friend :) . Just copied the SWF file in his directory and also working fine.

**Comment by BasilMas** [comment]
try the latest build

**Reply by bellejt** [reply]
pls give link so we can download the old version (independent) .There was an auto update from client this morning and not working anymore even online version with SWF file

**Reply by BasilMas** [reply]
use this manual

https://github.com/fedorovvl/tso_client/discussions/307#discussioncomment-7104244

**Comment by bellejt** [comment]
many THX.It is working

  _____  

Van: BasilMas ***@***.*** 
Verzonden: maandag 25 september 2023 20:41
Aan: fedorovvl/tso_client
CC: bellejt; Comment
Onderwerp: Re: [fedorovvl/tso_client] Test server connection impossible
since 05/09 TSO infrastructure maintenance (Discussion #307)



use this manual

#307 (reply in thread)
<https://github.com/fedorovvl/tso_client/discussions/307#discussioncomment-7
104244> 

-
Reply to this email directly, view
<https://github.com/fedorovvl/tso_client/discussions/307#discussioncomment-7
105055> it on GitHub, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/A4KTK44MYMBWVCW2TUHZTYLX4
HF2PANCNFSM6AAAAAA4WCMDBY> .
You are receiving this because you commented.
<https://github.com/notifications/beacon/A4KTK47N72UXF2YPFSSHFW3X4HF2PA5CNFS
M6AAAAAA4WCMDB2WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQANRVB6.gif> Message ID:
***@***.***>



-- 
Dit e-mailbericht is met Avast-antivirussoftware gecontroleerd op virussen.
www.avast.com

**Reply by bellejt** [reply]
and after update client not working again.keeps on starting

**Reply by BasilMas** [reply]
who has hammerman after infrastructure update - try to use test build from here
https://nightly.link/fedorovvl/tso_client/workflows/msbuild/master/client.zip

you can play without any chat rooms

**Reply by rsutherlandii** [reply]
playing now without the chat rooms...any idea when this chat room issue will be resolved?

**Reply by Oviz** [reply]
Thank you, wokrs well

**Comment by BasilMas** [comment]
> playing now without the chat rooms...any idea when this chat room issue will be resolved?

when all realms will update infrastructure and Sirris will fix chats

**Reply by rsutherlandii** [reply]
Thank you for the update and the hard work!
